main.nf

```diff
@@ -36,6 +36,7 @@ def helpMessage() {
       --star_index                  Path to STAR index
       --hisat2_index                Path to HiSAT2 index
       --fasta                       Path to Fasta reference
+      --transcriptome               Path to Fasta transcriptome
```

Comment:
Add `--aligner` in appropriate section here too.

Reply:
Done!